packages/lib/slots.ts

```diff
@@ -162,24 +162,23 @@ function buildSlotsWithDateRanges({
   offsetStart = offsetStart ? minimumOfOne(offsetStart) : 0;
   const slots: { time: Dayjs; userIds?: number[] }[] = [];
 
+  let interval = Number(process.env.NEXT_PUBLIC_AVAILABILITY_SCHEDULE_INTERVAL) || 1;
```

Comment:
Let a slot start at any minute if the frequency (timeslot interval) is not dividable by any of the defined starting times (60, 30, 20, 15, 10, 5). 

The main thing that will be changed is that the first slot of a date range from a 5 min event now doesn't need to start at 15 minute slot no more. This caused 'missing' slots, here is an example:
- 1:00, 1:05, 1:10, 1:15 is available 
- 1:00 gets booked
- With the old logic 1:15 would be the first available slot now
- With the new logic 1:05 is the first available slot

What also changes: 
- Slots with a timeslot interval (duration) of, for example, 7 minutes will now start at any arbitrary minute and not at a 15 minute slot.